components/tcf/secondLayer/src/components/ButtonsTriade.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState, useEffect} from 'react'
+import PropTypes from 'prop-types'
+import SuiButton from '@s-ui/react-atom-button'
+
+export default function ButtonsTriade({
+  isSaveButtonActive,
+  onSaveExitClick,
+  onAcceptAll,
+  onRejectAll
+}) {
+  const [saveButtonActive, setSaveButtonActive] = useState(false)
+  const checkSaveButton = () => isSaveButtonActive !== saveButtonActive
+
+  useEffect(() => {
+    checkSaveButton() && setSaveButtonActive(true)
+  }, [isSaveButtonActive])
```

Comment:
Remove the useEffect and put the isSaveButtonActive as initial state of saveButtonActive

Reply:
I can't because at the first rendering the isSaveButtonActive is false, but when a toggle is clicked, then the parent component changes the value of the isSaveButtonActive prop that is passing, but this ButtonsTriade doesn't get that chaneg of value, so I need the use effect to be able to notice that change of props